search/tests/test_sorting.py

```diff
@@ -0,0 +1,117 @@
+"""
```

Comment:
The meaning of this is to test the constants, since the strings need to match the index names, and the content of the lists returned.
If I do it in the interface, I need to check the content of the page and that would be much harder.